requirements.txt

```diff
@@ -0,0 +1 @@
+pyqtdarktheme
```

Comment:
@brgcode I forgot, for pip packages do we add it here or feedstock?

Reply:
we can't add pip installs to the feedstock. everything needs to be on conda-forge...

Reply:
if we add it here to the requirements file, at least it gets installed automatically with source installs...